sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -86,6 +105,28 @@ def shutdown(self) -> None:
         """
         self.storage.close()
 
+    # pylint: disable=protected-access
+    def _get_otel_resource_envelope(self, resource: Resource) -> TelemetryItem:
+        attributes = {}
+        if resource:
+            attributes = resource.attributes
+        envelope = _utils._create_telemetry_item(time_ns())
+        envelope.name = _METRIC_ENVELOPE_NAME
+        envelope.instrumentation_key = self._instrumentation_key
+        data_point = MetricDataPoint(
+            name=str("_OTELRESOURCE_")[:1024],
+            value=0,
+        )
+
+        data = MetricsData(
+            properties=attributes,
+            metrics=[data_point],
```

Comment:
Is this feature specifically only for the use case for heartbeat/detecting what azure compute the application is running in?

Reply:
Yeah. And supportability as well. I think they also generally want to bring in as much of OpenTelemetry as possible into non-otlp ingestion.